extension/parquet/include/parquet.json

```diff
@@ -1,138 +1,145 @@
 [
-  {
-    "class": "ParquetColumnDefinition",
-    "includes": [
-      "parquet_reader.hpp"
-    ],
-    "members": [
-      {
-        "id": 100,
-        "name": "field_id",
-        "type": "int32_t"
-      },
-      {
-        "id": 101,
-        "name": "name",
-        "type": "string"
-      },
-      {
-        "id": 103,
-        "name": "type",
-        "type": "LogicalType"
-      },
-      {
-        "id": 104,
-        "name": "default_value",
-        "type": "Value"
-      }
-    ],
-    "pointer_type": "none"
-  },
-  {
-    "class": "ParquetEncryptionConfig",
-    "includes": [
-      "parquet_crypto.hpp"
-    ],
-    "members": [
-      {
-        "id": 100,
-        "name": "footer_key",
-        "type": "string"
-      },
-      {
-        "id": 101,
-        "name": "column_keys",
-        "type": "unordered_map<string, string>"
-      }
-    ],
-    "pointer_type": "shared_ptr",
-    "constructor": ["$ClientContext"]
-  },
-  {
-    "class": "ParquetOptions",
-    "includes": [
-      "parquet_reader.hpp"
-    ],
-    "members": [
-      {
-        "id": 100,
-        "name": "binary_as_string",
-        "type": "bool"
-      },
-      {
-        "id": 101,
-        "name": "file_row_number",
-        "type": "bool"
-      },
-      {
-        "id": 102,
-        "name": "file_options",
-        "type": "MultiFileReaderOptions"
-      },
-      {
-        "id": 103,
-        "name": "schema",
-        "type": "vector<ParquetColumnDefinition>"
-      },
-      {
-        "id": 104,
-        "name": "encryption_config",
-        "type": "shared_ptr<ParquetEncryptionConfig>",
-        "default": "nullptr"
-      },
-      {
-        "id": 105,
-        "name": "debug_use_openssl",
-        "type": "bool",
-        "default": "true"
-      },
-      {
-        "id": 106,
-        "name": "explicit_cardinality",
-        "type": "idx_t",
-        "default": "0"
-      }
-    ],
-    "pointer_type": "none"
-  },
-  {
-    "class": "FieldID",
-    "includes": [
-      "parquet_writer.hpp"
-    ],
-    "members": [
-      {
-        "id": 100,
-        "name": "set",
-        "type": "bool"
-      },
-      {
-        "id": 101,
-        "name": "field_id",
-        "type": "int32_t"
-      },
-      {
-        "id": 102,
-        "name": "child_field_ids",
-        "type": "ChildFieldIDs"
-      }
-    ],
-    "pointer_type": "none"
-  },
-  {
-    "class": "ChildFieldIDs",
-    "includes": [
-      "parquet_writer.hpp"
-    ],
-    "members": [
-      {
-        "id": 100,
-        "name": "ids",
-        "type": "case_insensitive_map_t<FieldID>",
-        "serialize_property": "ids.operator*()",
-        "deserialize_property": "ids.operator*()"
-      }
-    ],
-    "pointer_type": "none"
-  }
-]
+	{
```

Comment:
Can we revert the formatting changes and keep only the actual changes to the file?